service/src/main/kotlin/io/provenance/api/domain/usecase/objectStore/store/StoreFile.kt

```diff
@@ -33,15 +35,15 @@ class StoreFile(
             storeObject.executeBlocking(
                 StoreObjectRequest(
                     if (!storeRawBytes)
-                        AssetOuterClassBuilders.Asset {
-                            idBuilder.value = id
-                            this.type = FileNFT.ASSET_TYPE
-                            description = file.filename()
-                            putKv(FileNFT.KEY_FILENAME, file.filename().toProtoAny())
-                            putKv(FileNFT.KEY_BYTES, bytes.toProtoAny())
-                            putKv(FileNFT.KEY_SIZE, bytes.size.toString().toProtoAny())
-                            putKv(FileNFT.KEY_CONTENT_TYPE, file.headers().contentType.toString().toProtoAny())
-                        }.toByteArray() else bytes,
+                        Asset.newBuilder().also {
+                            it.id = id.toUuid().toProtoUUID()
+                            it.type = FileNFT.ASSET_TYPE
+                            it.description = file.filename()
+                            it.putKv(FileNFT.KEY_FILENAME, file.filename().toProtoAny())
+                            it.putKv(FileNFT.KEY_BYTES, bytes.toProtoAny())
+                            it.putKv(FileNFT.KEY_SIZE, bytes.size.toString().toProtoAny())
+                            it.putKv(FileNFT.KEY_CONTENT_TYPE, file.headers().contentType.toString().toProtoAny())
+                        }.build().toByteArray() else bytes,
```

Comment:
FYI [you can still use the builder syntax if you like, it just uses slightly different syntax now](https://github.com/provenance-io/metadata-asset-model/blob/363be0f729a15d0d940404f25b83eaed1cbde20a/src/test/kotlin/tech/figure/asset/v1beta1/AssetTest.kt#L19-L27)

Reply:
:nice: might as well stick with what i put